packages/ui/src/assets/svg/icons/switch-arrows.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
maybe rename it to `arrows-switch`? for grouping icons by name

Reply:
then better will be `arrows-bidirectional` imo, because all other arrows named like `arrow-like`, `arrow-left`